src/mjan.c

```diff
@@ -300,18 +329,95 @@ void DebugMenu(void) {}
 /*********** BEGIN VARIANT FUNCTIONS ***********/
 
 // as of this version, only standard exists
-int NumberOfOptions(void) {
-    return 1;
+int NumberOfOptions() {
+	return 2*3;
 }
-int getOption(void) {
-  return 0;
+
+int getOption() {
+	int option = 1;
+	if(gStandardGame) option += 1;
+	if(allDiag) option += 1 *2;
+	if(mainDiag) option += 2 *2;
+
+	return option;
+}
+
+STRING GetVarString() {
+	switch (getOption())
+	{
+	case 1:
+		return "Mis�re game with standard moves";
```

Comment:
what is the unknown character?

Reply:
I think it's a fancy french e, but I've removed it.